safesql/src/main/java/org/dhatim/safesql/SafeSqlAppendable.java

```diff
@@ -0,0 +1,70 @@
+package org.dhatim.safesql;
+
+import java.util.stream.Stream;
+
+public interface SafeSqlAppendable {
+
+    SafeSqlAppendable param(int num);
+    SafeSqlAppendable param(long num);
+    SafeSqlAppendable param(double num);
+    SafeSqlAppendable param(boolean bool);
+    SafeSqlAppendable param(Object obj);
+    SafeSqlAppendable params(Object param1, Object param2);
+    SafeSqlAppendable params(Object param1, Object param2, Object param3);
+    SafeSqlAppendable params(Object param1, Object param2, Object param3, Object... others);
+    SafeSqlAppendable params(Object... parameters);
```

Comment:
Yeah, in fact I see in [JDK9 Set.java](http://hg.openjdk.java.net/jdk9/jdk9/jdk/file/6e4ff59afb5d/src/java.base/share/classes/java/util/Set.java#l482) or in [Guava ImmutableSet.java](https://google.github.io/guava/releases/19.0/api/docs/com/google/common/collect/ImmutableSet.html) that they use this pattern. I suppose it is for speed. Method with one, two or three parameters cover the 90% of uses and they are faster than the array one.